### ISSUE TITLE
F.cross_entropy() got an unexpected keyword argument "label_smoothing"

### ISSUE THREAD

**Issue by prevso1088** [ShiqiYu/OpenGait#123]
## System information (version)

- Pytorch => 1.6.0
- Operating System / Platform => Ubuntu 20.04
- Cuda => 10.2

## Detailed description

Commit 5c61842 uses "label_smoothing" as a keyword argument for F.cross_entropy() but this keyword argument was not introduced until PyTorch 1.10, causing errors for lower versions. It might be good to maintain backward compatibility or update environment requirements.

## Steps to reproduce

Train any model with the cross entropy loss.

## Issue submission checklist
 - [x] I checked the problem with the documentation, FAQ, and issues, and have not found a solution.

**Comment by darkliang** [comment]
Thanks for reporting this issue. I will fix it as soon as possible.
